docs/quick-start.mdx

```diff
@@ -4,134 +4,339 @@ title: Quick Start
- it can be integrated into the Build scripts of a Continuous Integration system.
+## Installing Pester
 
-Pester contains a powerful set of Mocking Functions that allow tests to mimic and mock the
-functionality of any command inside of a piece of PowerShell code being tested. See
-[Mocking with Pester](./usage/mocking).
+To install Pester it is usually enough to just do `Install-Module Pester -Force`. And then follow it by  
+`Import-Module Pester -PassThru`. This is the output you should see in the console: 
 
-Pester can produce artifacts such as Test Results and can be used for Generating
-[Code Coverage Metrics](./usage/code-coverage) metrics output files which can be used
-for reporting purposes and in build pipelines. See [Showing Test-Results in CI](./usage/test-results).
+```shell
+C:\> Import-Module Pester -Passthru
+
+ModuleType Version    PreRelease Name
+---------- -------    ---------- ----
+Script     5.0.4                 Pester
+```
+
+Full installation guide is available in [installation](introduction/installation).
 
 ## Creating a Pester Test
 
-To start using Pester, use the optional [New-Fixture](./commands/New-Fixture) function to scaffold
-both a new implementation function and a test function. New-Fixture assumes that you will be testing
-the contents of a `.ps1` file (not a `.psm1` file, or Script Module), that the test script should be
-in the same directory as the script under test, and that the name of the test script should be
-`<Name of Script Under Test>.Tests.ps1`.
+To start using Pester, create a new file called `Get-Planet.Tests.ps1`. `Get-Planet` is the name of the function
+we will be testing. Feel free to replace that with your own function name. The file name is important because Pester 
+uses a naming convention, all `*.Tests.ps1` files will be inspected for tests.
 
-To scaffold a new project, use the script below to generate a placeholder function script and test script.
+Inside of the file paste this code:
 
 ```powershell
-New-Fixture deploy Clean
+BeforeAll { 
+    function Get-Planet ([string]$Name = '*') {
+        $planets = @(
+            @{ Name = 'Mercury' }
+            @{ Name = 'Venus'   }
+            @{ Name = 'Earth'   }
+            @{ Name = 'Mars'    }
+            @{ Name = 'Jupiter' }
+            @{ Name = 'Saturn'  }
+            @{ Name = 'Uranus'  }
+            @{ Name = 'Neptune' }
+        ) | ForEach-Object { [PSCustomObject] $_ }
+
+        $planets | Where-Object { $_.Name -like $Name }
+    }
+}
 
-<#
-Creates two files:
-./deploy/Clean.ps1
-#>
+Describe 'Get-Planet' {
+    It 'Given no parameters, it lists all 8 planets' {
+        $allPlanets = Get-Planet
+        $allPlanets.Count | Should -Be 8
+    }
+}
+```
 
-function Clean {
+This code uses multiple Pester keywords, and we will go over them in detail soon, but for now let's just run it.
 
-}
+In your console run `Invoke-Pester -Output Detailed C:\t\Planets\Get-Planet.Tests.ps1`:
+
+```shell
```

Comment:
Is this `shell` by intend ?

Reply:
Yes, it's highlight for shell output, not a misspelled powershell. Does it not look good? Am I not consistent with using it for all shell output?

Reply:
Looks like there is a difference between the highlihter my blog and github use, and prism that docs use. Replaced with ` ```shell-session`. Could also just use ` ``` `, because it does not highlight, but I like to show that not higlighting is intentional.

Reply:
OK, I thought all command line examples were consistently using ` ```powershell` thus far (which would make sense to me) but any highlighter is fine by me.

Reply:
These were command output examples. The powershell highlighting there made no sense.

Reply:
Yeah, you are right. I did not read all content properly enough, will do next time. You did a great job on the page though, it reads great.

Reply:
Thanks 🙂